judge/migrations/0111_blank_assignees_ticket.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.15 on 2020-09-13 16:05
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0110_default_output_prefix_override'),
+    ]
+
+    operations = [
+         migrations.AlterField(
```

Comment:
Indentation is wrong here, did you try to edit this migration manually?

Reply:
I did edit this manually, and accidentally left an extra space 🤦‍♂️

Should be fixed now.